api/v1alpha1/constant.go

```diff
@@ -33,3 +33,11 @@ const (
 	// DefaultMode is exclusive
 	DefaultMode PlacementMode = ""
 )
+
+type OperationType string
+
+const (
+	AddLabel    OperationType = "Add"
+	DeleteLabel OperationType = "Delete"
+	UpdateLabel OperationType = "UpdateValue"
```

Comment:
`Add` and `UpdateValue` are somewhat confused

Reply:
I don't think definition of OperationType should be placed in api package